.github/workflows/check_new_commits.yml

```diff
@@ -35,5 +43,4 @@ jobs:
         uses: benc-uk/workflow-dispatch@v1.2
         with:
           workflow: "Ghidra Build"
-          token: ${{secrets.PAT_TOKEN}}
```

Comment:
Oh, the personal access token isn't necessary anymore? nice!

Reply:
The `github.token` passed to the action by default appears to work just fine for this! Wasn't sure if there was another reason it was using a PAT